libr/bin/format/elf/elf.c

```diff
@@ -2660,35 +2660,41 @@ static size_t get_num_relocs_sections(ELFOBJ *bin) {
 }
 
 static size_t get_num_relocs_approx(ELFOBJ *bin) {
-	return get_num_relocs_dynamic (bin) + get_num_relocs_sections (bin);
+	size_t rd = get_num_relocs_dynamic (bin);
```

Comment:
is it really necessary to split?

Reply:
imho it's more readable but not necessary. just a matter of taste

Reply:
seems a bit too split... we have much longer lines than this and this imo looks quite readable already
`get_num_relocs_dynamic (bin) + get_num_relocs_sections (bin)`. If this is not ok, I guess that any sort of operation that does not involve super simple type is not.

Reply:
i can rollback if you like, i prefer to read in vertical